ui/raidboss/raidboss.js

```diff
@@ -52,6 +54,12 @@ UserConfig.getUserConfigLocation('raidboss', function(e) {
       console.log('Enabling audio via query parameter');
   }
 
+  let PlayerNameOverride = params.get('player');
+  if (PlayerNameOverride !== null) {
+    Options.PlayerNameOverride = PlayerNameOverride;
+    console.log('Enabling player name override via query parameter, ' + PlayerNameOverride);
```

Comment:
Sorry for any bad communication on my part.  What I wanted was to set `Options.BrowserTTS = false` here, and to have the line in the PopupText constructor just say `if (this.options.BrowserTTS) {`, so-as to not couple these concepts so closely.  Doing it here will always override any config or user javascript file.

Reply:
Will setting `Options.BrowserTTS` here persist that change to the config file, or is it purely ephemeral?

Reply:
Yeah, it's ephemeral.  Only changes on the config page itself get persisted to the json file.  Other settings (at runtime, or in user javascript files), are just temporary overrides.